simple_history/tests/tests/test_models.py

```diff
@@ -1478,3 +1482,21 @@ def test_revert(self):
 
         self.one = Street.objects.get(pk=id)
         self.assertEqual(self.one.history.count(), 4)
+
+
+@override_settings(**database_router_override_settings)
+class UsingSeparateDatabaseTestCase(TestCase):
+    multi_db = True
+    db_name = "other"
+
+    def setUp(self):
+        self.user = get_user_model().objects.create(
+            username="username", email="username@test.com", password="top_secret"
+        )
+        self.model_ = ModelWithHistoryInDifferentDb(name="test")
+
+    def tearDown(self):
+        self.model_ = None
+
+    def test_using_separate_db(self):
```

Comment:
this doesn't test that `using` gets used though. Can we test for that? 
```
def test_history_saves_to_database_specified_in_using(self):
    self.model_ = ModelWithHistoryInDifferentDb(name="test")
    self.model_.save()
    from simple_history.tests.tests.utils import HistoricalModelWithHistoryInDifferentDb
    self.assertEqual(1, HistoricalModelWithHistoryInDifferentDb.objects.using('other').count())
```
Which brings up another question: if we do this, does querying break since using is only specified when writing to the database?

Reply:
Looks like we're good on both accounts -- I've updated the tests.

Reply:
Oh, I figured out the problem. `ModelWithHistoryInDifferentDb` is now in `other` as well, so both tables in the same ("other") database, so these tests aren't really testing anything. I think the queries will likely fail if the base model is on a separate database. I'm thinking my preference would be to scrap the `self.using` variable, and instead have a `inherit_db` parameter that defaults to `True`. If `inherit_db` is false, then no `using` parameter is passed to `create_historical_record` and the history table relies entirely on routers. We could then add a using parameter afterward, but I think the first step needs to be allowing the routers to work correctly. Otherwise, you need to specify `using` everywhere, including the manager (so that the queryset is correct), which seems more difficult and less clean

Reply:
Oh, i see what you mean. This is the case because I've marked `app_label = "external"`. I had to do this because the TestDbRouter was not being used.

I am currently using this branch and can verify that it is possible to store the operational model separately from the historical one and also query both. I think the root issue here is how the TestDbRouter is used.

I'm not sure what the solution involving `inherit_db` looks like.